owasp-top10-2017-apps/a5/ecommerce-api/docs/ATTACK.md

```diff
@@ -0,0 +1,45 @@
+# Attack Narrative - Ecommerce API
+The main goal of this document is to describe how a malicious user could exploit a lack of access control to obtain tokens that he should not have access to.
```

Comment:
I think it should be a good idea to explicit state here that we are exploiting our app from SecDevLabs. 

`The main goal of this documentation is to describe how a malicious user could exploit a Broken Access Control vulnerability intentionally installed on Vulnerable Ecommerce API from secDevLabs.`

Reply:
Thanks @rafaveira3 ! I'll add that!